### PR TITLE
FABGW-8: Fix race condition in Finder unit test

### PR DESCRIPTION
**Pull request by bestbeforetoday**
Keep sending messages on the send channel to ensure there is still a message available to read once the listener is established.

### CODE REVIEW

File: internal/pkg/gateway/commit/finder_test.go [hunk 1]
```diff
@@ -23,6 +23,21 @@ type queryProvider interface { // Mimic QueryProvider to avoid circular import w
 }
 
 func TestFinder(t *testing.T) {
+	sendUntilDone := func(commitSend chan<- *ledger.CommitNotification, msg *ledger.CommitNotification) chan struct{} {
+		done := make(chan struct{})
+
+		go func() {
+			for {
+				select {
+				case commitSend <- msg:
```
Comment (jyellick) [inline internal/pkg/gateway/commit/finder_test.go:33]: This will result in a tight loop, no?  Probably worth putting at least a small sleep in here? 10ms?

Comment (bestbeforetoday) [reply internal/pkg/gateway/commit/finder_test.go:33]: 👍

**Review by jyellick — APPROVED** [review]
Not my favorite approach, but since I don't have another and it's a flake that could be hitting others, merging.

**Pull request merged**